configure.ac

```diff
@@ -558,6 +558,7 @@ src/Makefile
 src/cli/Makefile
 src/ncp/Makefile
 src/core/Makefile
+src/diag/Makefile
```

Comment:
@darconeous, yes, it was just added :)

Thank you @chshu for addressing all the comments.